requirements.txt

```diff
@@ -17,3 +17,4 @@ defusedxml>=0.4
 django-appconf>=1.0
 user-agents>=1.1.0
 filelock>=3.0.1
+openpyxl>=2.5.0
```

Comment:
I'd prefer this to be optional dependency and the xlsx to be available only if it's installed.

Reply:
sorry, I do not have a clue how to implement optional dependencies in python

Reply:
What would be needed for this:

- Move the dependency to the requirements-optional.txt
- Make the import of `openpyxl` conditional, register the excel format only if it is available

However I can do this later if other issues are addressed.